alvr/xtask/src/format.rs

```diff
@@ -0,0 +1,77 @@
+use alvr_filesystem as afs;
+use std::fs;
+use walkdir::{DirEntry, WalkDir};
+use xshell::{cmd, Shell};
+
+fn should_format(entry: &DirEntry) -> bool {
+    if entry.file_type().is_dir() {
+        match entry.path().file_name().unwrap().to_str().unwrap() {
+            "shared" | "include" => false,
+            _ => true,
+        }
+    } else {
+        entry
+            .path()
+            .extension()
+            .is_some_and(|ext| match ext.to_str().unwrap() {
+                "c" | "cpp" | "h" | "hpp" => true,
+                _ => false,
+            })
```

Comment:
I think you could use `matches!()` instead, or `["your", "matches"].contains(name)`. same for the expression below